frontend/packages/dev-console/src/components/topology2/filters/filter-utils.ts

```diff
@@ -0,0 +1,27 @@
+export enum ShowFiltersKeyValue {
+  podCount = 'Pod Count',
+  setTraffic = 'Set Traffic',
+  eventSources = 'Event Sources',
+}
+
+export enum ExpandFiltersKeyValue {
+  knativeServices = 'Knative Services',
+  appGrouping = 'Application Groupings',
+  operatorGrouping = 'Operator Groupings',
+}
+
+export type TopologyFilters = {
+  display: DisplayFilters;
+  searchQuery: SearchFilter;
```

Comment:
nit: we can have similar in key-value i.e `searchQuery: SearchQuery`

Reply:
fixed in https://github.com/openshift/console/pull/3746/commits/2cb083eeccb662abb5c2ff9b4630a026333c3dd1